simapp/app.go

```diff
@@ -120,7 +120,7 @@ type SimApp struct {
 	CrisisKeeper     crisis.Keeper
 	UpgradeKeeper    upgrade.Keeper
 	ParamsKeeper     params.Keeper
-	IBCKeeper        ibc.Keeper
+	IBCKeeper        *ibc.Keeper
```

Comment:
IBC Keeper needed to be changed to a pointer so that we could `SetRouter` correctly

Reply:
can you add that comment on the code? ^^